containers/ecr-viewer/.env.sample

```diff
@@ -9,18 +9,27 @@ AZURE_CONTAINER_NAME=ecr-viewer-files
 
 # AWS Connection
 AWS_REGION=us-east-1
-AWS_CUSTOM_ENDPOINT=http://localhost:4566
 ECR_BUCKET_NAME=ecr-viewer-files
 AWS_ACCESS_KEY_ID=123
 AWS_SECRET_ACCESS_KEY=xyz
+# Only for local use
+AWS_CUSTOM_ENDPOINT=http://localhost:4566
+
+# GCP GCS
+GCS_BUCKET_NAME=ecr-viewer-files
+# Only required if not using Application Default Credentials
+GCP_KEY='{ "client_email": "ecr@example.com", "private_key": "1234" }'
+GCP_PROJECT_ID=projectId
+# Only for local use
+GCS_API_ENDPOINT=http://localhost:10001/
 
 # Postgres
 DATABASE_URL=postgres://postgres:pw@localhost:5432/ecr_viewer_db
 
 # SQL Server
 SQL_SERVER_USER=sa
 SQL_SERVER_PASSWORD=Password1!
-# SQL_SERVER_HOST=sqlserver
+SQL_SERVER_HOST=localhost
```

Comment:
why un-comment?

Reply:
This doesn't matter with the kysley stuff anyways so I'll remove to reduce conflicts. This should have been uncommented during the migration to using local network for ecr-viewer in docker compose since it is now the same value as in docker-compose.